internal/envconfig/envconfig.go

```diff
@@ -55,8 +55,11 @@ var (
 	// setting the environment variable "GRPC_EXPERIMENTAL_ENABLE_NEW_PICK_FIRST"
 	// to "true".
 	NewPickFirstEnabled = boolFromEnv("GRPC_EXPERIMENTAL_ENABLE_NEW_PICK_FIRST", false)
-	//HTTPProxy is set to empty string if HTTPS_PROXY env it is not set.
-	HTTPProxy = stringFromEnv("HTTPS_PROXY", "")
+	// HTTPSProxy indicates the HTTPS_PPROXY environment which is set when proxy
```

Comment:
this is probably too much repetition. Also, there are some typos. I think it can be simple "HTTPSProxy retrieves the HTTPS proxy address from the environment variable 'HTTPS_PROXY'. If the variable is not set, its set to empty string. "

Reply:
Done.